Features/Users/users.feature

```diff
@@ -67,3 +67,12 @@ Feature: Use the eZ Users field
         And I fill in "Password" with "12345"
         And I fill in "Confirm password" with "123456"
         Then I should see error messages
+
+    @javascript @edge
+    Scenario: Validate "Send to trash" button is replaced with "Delete" when browsing users
+    Scenario: Validate that users have the "Delete" button available
```

Comment:
I was really suggesting that the sentence was _replaced_. As I see it, the scenario covers the fact that "Send to trash" is replaced with "Delete". We can easily formulate that as one sentence. Or did you have something else in mind ?